src/services/gnosis/errors/OperatorError.ts

```diff
@@ -63,9 +63,10 @@ export default class OperatorError extends Error {
     try {
       console.log(orderPostError);
       if (orderPostError.errorType) {
-        const errorMessage = i18n.global.t(
-          `apiErrorCodeDetails.${orderPostError.errorType.toString()}`
-        );
+        const errorMessage = i18n.global
+          .t(`apiErrorCodeDetails.${orderPostError.errorType.toString()}`)
+          .replace('apiErrorCodeDetails.', ''); // if a translation cannot be found, keep only the error code
```

Comment:
`errorMessage` prob won't be empty anymore since failed string checks return the path - so in order to pass on just the errorCode (for potential custom error handling) - strip it.